chef-client-run.go

```diff
@@ -78,6 +78,13 @@ func chefClientRun(nodeClient chef.Client, nodeName string, runList []string, ge
 
 	time.Sleep(time.Duration(sleepDuration) * time.Second)
 
+	// Ensure that what we post at the end of the run is different frm previous nodes
```

Comment:
Can you please fix the typo and also make this say "from previous runs"? That makes more sense to me anyway.